docs/components/accordion.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.6.0/cjs/popper.min.js"></script> -->
+        <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+        <script src="/cocoandeve-styleguides/js/main.js"></script>
+    </head>
+    <body>
+        <div class="container mt-4">
+            <h1>Accordion</h1>
+
+            <div class="row">
+                <ul class="accordion list-unstyled col-9 accordion--menu py-2" id="accordionOffCanvas" role="tablist" aria-multiselectable="true">
+                    <li class="card card--range py-1">
+                        <div class="card-header card-header--media p-0" role="tab" id="headingTwo1">
```

Comment:
accordion component should be very simple, not the one used in the menu.
we need to develop a basic simple accordion that can be used everywhere in the website, starting from the basic one from bootstrap.
check also for other accordion uses across the website, find common styles and create a basic version. All accordions usually have a title, contents and a side icon